code/drasil-website/lib/Drasil/Website/Example.hs

```diff
@@ -221,11 +234,18 @@ getCodePath path ex programLang = path ++ "code/stable/" ++ map toLower ex ++ "/
 -- | Uses 'exRt' path (srsDoxPath in this module).
 getDoxPath path ex programLang = path ++ map toLower ex ++ "/doxygen/" ++ programLang ++ "/index.html" -- need example path
 
+-- | Get the paths for the source code locations
```

Comment:
The name of the function and the comment don't give me any clue that this is related to `drasil-example`, and it should. Also, we should try to not hard-code so much, so we should be constructing "code/drasil-example/" from other piecees.